charts/node-analyzer/templates/clusterrole-node-analyzer.yaml

```diff
@@ -33,19 +33,6 @@ rules:
     - "flowschemas"
   verbs:
     - list
-- apiGroups:
```

Comment:
@a1ex-var1amov we still have on-premise customers using the Benchmark Runner, can we add `{{- if (include "nodeAnalyzer.deployBenchmarkRunner" .) }}` for add those additional permission only when `benchmarkRunner.deploy` is set to `true` ?

Reply:
Thanks Alex for fixing it